src/views/ModelPlan/TaskList/index.tsx

```diff
@@ -68,7 +70,18 @@ const TaskList = () => {
   const taskListItemStatus = (key: string) => {
     switch (key) {
       case 'basics':
-        return basics === null ? 'READY' : 'IN_PROGRESS';
+        if (basics === null) {
+          return 'READY';
+        }
+        if (
+          basics?.status === 'COMPLETE' &&
+          milestones?.status === 'COMPLETE'
+        ) {
+          return 'COMPLETE';
+        }
+        return 'IN_PROGRESS';
```

Comment:
On second thought, this logic probably shouldn't be here. 

@ClayBenson94 @StevenWadeOddball 
Will there be a field in the BE that I can use to determine when the section is `READY`, `IN PROGRESS` or `COMPLETE`

cc @patrickseguraoddball